playwright/tests/page-objects/overview.ts

```diff
@@ -170,10 +172,8 @@ export class Overview {
   async hasGroup(name: string, tagLabel?: string, numOfNewCards?: number) {
     const { group, trigger } = await this.group(name, tagLabel);
     await expect(trigger).toHaveAttribute('data-state', 'open');
-    const nestedArtifacts = group.locator('.artifact-card');
-    expect(await nestedArtifacts.count()).toBeGreaterThan(0);
     const nestedNew = group.locator('.new-artifact-card');
-    expect(await nestedNew.count()).toBe(numOfNewCards ?? 1);
+    expect(await nestedNew.count()).toBe(numOfNewCards ?? 0);
```

Comment:
not your fault, but this wrong.
The problem is here if the "assertion" is not right yet but also the UI is maybe still doing something, the test will not retry after some time. 
Because here it gets the count of this new cards and then test if it is equal to a given number.

But it should be written like the following:
`await expect(nestedNew).toHaveCount(numOfNewCards)`